starthost-plugin/src/main/java/br/com/autonomiccs/starthost/systemVm/AutonomiccsStartHostSystemVmManager.java

```diff
@@ -357,24 +347,7 @@ public void afterPropertiesSet() throws Exception {
     }
 
     private void copyStartHostApplicationToTempFile() throws FileNotFoundException, IOException {
```

Comment:
this method does not make any sense anymore. You can simple do s`tartHostApplicationExecutable = AutonomiccsJarAbstraction.WAKEONLAN.getJar()` at the variable declaration.